mlflow/server/handlers.py

```diff
@@ -163,6 +164,18 @@ def get_artifact_handler():
         return send_file(filename, as_attachment=True)
 
 
+@catch_mlflow_exception
+def get_versions_handler():
+    response_message = {
+        'mlflow': pkg_resources.get_distribution("mlflow").version,
+        'mlflow-criteo': os.getenv('VERSION', 'UNKNOWN'),
```

Comment:
I'm not sure it is a good idea to keep adding criteo specific stuff in the fork. Maybe we should add this endpoint in mlflow-criteo, what do you think ?

Reply:
It it's possible to do so, then yes !